server/block/stone_wall_type.go

```diff
@@ -0,0 +1,127 @@
+package block
+
+// StoneWallType represents a type of cobblestone wall.
+type StoneWallType struct {
```

Comment:
I think we should have walls that match the way we structure blocks, with for example SandstoneWall{red bool}, CobblestoneWall{mossy bool} etc. It's much more convenient to convert a block to its respective wall that way. Maybe we could re-use code using generics somehow?